css/production/mail.css

```diff
@@ -1,56 +1,99 @@
 #page-wrapper {
-  max-width: 100%;
-  padding: 5px; }
-  #page-wrapper .col-lg-12 {
+    max-width: 100%;
+    padding: 5px;
+}
+
+#page-wrapper .col-lg-12 {
     padding: 0 !important;
-    max-width: 100%; }
-  #page-wrapper .row {
-    margin: 0; }
-  #page-wrapper #mailTable th {
+    max-width: 100%;
+}
+
+#page-wrapper .row {
+    margin: 0;
+}
+
+#page-wrapper #mailTable th {
     max-width: 100px;
-    min-width: 100px; }
-    #page-wrapper #mailTable th .subject {
-      max-width: 80%; }
-  #page-wrapper #mailTable td {
-    max-height: 500px; }
-    #page-wrapper #mailTable td iframe {
-      width: 100%;
-      height: 100%; }
-  #page-wrapper #mailTable:hover {
-    cursor: default !important; }
+    min-width: 100px;
+}
+
+#page-wrapper #mailTable th .subject {
+    max-width: 80%;
+}
+
+#page-wrapper #mailTable td {
+    max-height: 500px;
+}
+
```

Comment:
Släng in det i scssen (annars skrivs det över vid gulp test eller gulp build), och det blir lite mer lättläsligt när det är så nästlad css.